READMEs/dfbuyer.md

```diff
@@ -0,0 +1,63 @@
+<!--
```

Comment:
This README needs to be linked from the main README.md

Reply:
Good idea. Call it "OPF bots" and put it between "Run bots" and "Develop predictoor" sections.